tests/foreman/cli/test_ldapauthsource.py

```diff
@@ -323,6 +323,48 @@ def test_usergroup_with_usergroup_sync(self, ipa_data):
 
 
 @run_in_one_thread
+class TestOpenLdapAuthSource:
+    """Implements OpenLDAP Auth Source tests in CLI"""
+
+    @tier2
+    @pytest.mark.parametrize('server_name', **parametrized(generate_strings_list()))
+    @upgrade
+    def test_positive_end_to_end_with_open_ldap(self, open_ldap_data, server_name):
+        """CRUD LDAP Operations with OpenLDAP
+
+        :id: f84db334-0189-11eb-846c-d46d6dd3b5b2
+
+        :expectedresults: Whether creating/updating/deleting LDAP Auth with OpenLDAP is successful.
+
+        :CaseImportance: High
+        """
+        auth = make_ldap_auth_source(
+            {
+                'name': server_name,
+                'onthefly-register': 'true',
+                'host': open_ldap_data['ldap_hostname'],
+                'server-type': LDAP_SERVER_TYPE['CLI']['posix'],
+                'attr-login': LDAP_ATTR['login_ad'],
+                'attr-firstname': LDAP_ATTR['firstname'],
+                'attr-lastname': LDAP_ATTR['surname'],
+                'attr-mail': LDAP_ATTR['mail'],
+                'account': open_ldap_data['ldap_user_name'],
+                'account-password': open_ldap_data['ldap_user_passwd'],
+                'base-dn': open_ldap_data['base_dn'],
```

Comment:
With the change to the fixture, these should change as well. Can you also run the tests again?

Reply:
missed that thanks